bindings/nodejs/examples/how_tos/advanced_transactions/advanced_transaction.ts

```diff
@@ -36,9 +35,12 @@ async function run() {
         // To sign a transaction we need to unlock stronghold.
         await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
 
-        // Create an ouput with amount 1_000_000 and a timelock of 1 hour
-        const in_an_hour = Math.floor(Date.now() / 1000) + 3600;
-        const basicOutput = await new Client({}).buildBasicOutput({
+        const client = await wallet.getClient();
+
+        // Create an output with amount 1_000_000 and a timelock of 1 hour
+        // TODO !!!
```

Comment:
What is the TODO?

Reply:
get a proper timeslot, its the same todo as in thibaults pr. need some methods to calculate this

Reply:
Can you add that to the comment?